tools/clang/test/DXILValidation/compute_node_compatibility.hlsl

```diff
@@ -1,36 +1,84 @@
-// RUN: %dxc -T lib_6_8 %s 
 // ==================================================================
-// Check that validation errors are generated when both compute and
-// node are specified and node input or outputs are also present.
-// The validation test will add compute to each of the following
-// shaders in turn, and check for the expected error message.
-// We also check that only Broadcasting nodes may be used with
-// compute.
+// Errors are expected for shaders with both "node" and "compute"
+// specified when:
+// - the launch type is not Broadcasting
+// - a broadcasting node has an input record and/or output records
+// This test operates by changing the [Shader(node)] metadata entry
+// to [Shader(compute)] as if both had been specified in the HLSL,
+// for each shader in turn.
 // ==================================================================
 
-struct RECORD {
- uint a;
+struct RECORD
+{
```

Comment:
This formatting change goes from LLVM-style to... something else. It would be better to be LLVM-style even in our tests.

Reply:
That was unintentional - fixed.